mlfinlab/bet_sizing/targeting.py

```diff
@@ -0,0 +1,19 @@
+"""
+This module contains functionality for determining bet sizes for investments based on volatility targeting.
+"""
+
+import numpy as np
+
+
+def volatility_targeting(returns: np.ndarray, target_vol: float, annualize_factor: int):
+    """
+    Calculate the position size to make the expected volatility of the portfolio be the target variability.
+
+    :param returns: (np.ndarray) This is series of returns of your strategy or a asset.
+    :param target_vol: (float) volatility(%) that you target.
+    :return: (float) position size you should use to change the volatility to be target volatility.
+    """
```

Comment:
Please add the `annualize_factor` variable to docstring description.

Reply:
This is my fault. I will add it.